### PR TITLE
lesson/Added video component

### PR DESCRIPTION
**Pull request by mrfinansist84**

### CODE REVIEW

File: ui/src/containers/LessonCreator/components/Widget/Widget.tsx [hunk 1]
```diff
@@ -1,46 +1,71 @@
...
-import { Text, Video, Picture } from '../';
+import { useState } from 'react';
+import { Video } from '../';
 import './Widget.scss';
 
 interface IProps {
   deleteItem: (id: string) => void;
   element: any;
   elementId: string;
+  handleElementChange: (state: any, elementId: string) => void;
```
Comment (stas-veriho) [inline ui/src/containers/LessonCreator/components/Widget/Widget.tsx:11]: Function usually should be after variables

File: ui/src/containers/LessonCreator/components/Widget/Widget.tsx [hunk 2]
```diff
@@ -1,46 +1,71 @@
...
+    })
+  };
 
   const getElement = () => {
     let result;
 
-    if (element.type === 'text') {
-      result = <Text {...element} />;
-    }
+    // if (element.type === 'text') {
```
Comment (stas-veriho) [inline ui/src/containers/LessonCreator/components/Widget/Widget.tsx:33]: Remove comment

File: ui/src/containers/LessonCreator/components/Widget/Widget.tsx [hunk 3]
```diff
@@ -1,46 +1,71 @@
...
+        handleChange={handleChange}
+        handleElementChange={handleElementChange}
+        />
+      )
+    };
 
-    if (element.type === 'picture') {
-      result = <Picture {...element} />;
-    }
+    // if (element.type === 'picture') {
```
Comment (stas-veriho) [inline ui/src/containers/LessonCreator/components/Widget/Widget.tsx:47]: Remove comment

**Comment by mrfinansist84** [comment]
need updates

**Pull request closed without merging**